### PR TITLE
fixes #159

### LINKED ISSUE #159: re-initialize arguments for each resource
Need to support changing the window size based on orientation for each resource when multiple are provided

### PR DESCRIPTION
**Pull request by codecounselor**

### CODE REVIEW

File: lib/exportJob.js [hunk 1]
```diff
@@ -79,31 +81,28 @@ class ExportJob extends EventEmitter {
...
 
     // TODO: Check for different domains, this is meant to support only a single origin
     const firstUrl = this.input[0]
-    this._setSessionCookies(args.cookies, firstUrl, win.webContents.session.cookies)
+    this._setSessionCookies(this.args.cookies, firstUrl, win.webContents.session.cookies)
 
     const windowEvents = []
     // The same listeners can be used for each resource
-    this.passThroughEvents(win, eventPrefix)
+    this._passThroughEvents(win, eventPrefix)
```
Comment (codecounselor) [inline lib/exportJob.js]: this should have been private all along

File: lib/exportJob.js [hunk 2]
```diff
@@ -79,31 +81,28 @@ class ExportJob extends EventEmitter {
...
-    const args = this.args
 
-    const win = this._launchBrowserWindow(args)
+    const win = this._launchBrowserWindow()
     this.window = win
 
     // TODO: Check for different domains, this is meant to support only a single origin
     const firstUrl = this.input[0]
-    this._setSessionCookies(args.cookies, firstUrl, win.webContents.session.cookies)
+    this._setSessionCookies(this.args.cookies, firstUrl, win.webContents.session.cookies)
```
Comment (codecounselor) [inline lib/exportJob.js]: being consistent about just using this.args everywhere

File: lib/exportJob.js [hunk 3]
```diff
@@ -350,9 +378,9 @@ class ExportJob extends EventEmitter {
       }
     } else {
       pageDim = pageDimensions[pageSize]
+      // Flip if landscape
+      pageDim = landscape ? {x: pageDim.y, y: pageDim.x} : pageDim
```
Comment (codecounselor) [inline lib/exportJob.js]: Orientation should never be changed if someone specified custom dimensions, the landscape setting will just be ignored in this scenario.  The browser window size and the PDF size ought to be dictated only by the pageSize object.

Comment (mattsgarlata) [inline lib/exportJob.js]: I don't totally follow what's going on here, but would this be helpful to include as a comment inside the code?

Comment (codecounselor) [inline lib/exportJob.js]: I updated the test instead, it was technically a bug so I'd rather let the test document the expected behavior.  Thanks for asking.

**Pull request merged**